controllers/csi/gc/logs.go

```diff
@@ -0,0 +1,124 @@
+package csigc
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	dtcsi "github.com/Dynatrace/dynatrace-operator/controllers/csi"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+)
+
+const (
+	maxLogFolderSize    = 300000
```

Comment:
Suggest to add a suffix: `maxLogFolderSizeBytes`

Reply:
Renamed